docs/setup/install.md

```diff
@@ -30,7 +30,7 @@ machine type for your nodes.**
 To install a released version of Jobset in your cluster, run the following command:
 
 ```shell
-VERSION=v0.3.1
+VERSION=v0.3.2
```

Comment:
can we define this as a variable in md language instead of having to update it in multiple places?